en/views.rst

```diff
@@ -460,7 +462,7 @@ Contacts plugin::
     {
         public function view_active()
         {
-            $this->layout = 'Contacts.contact';
+            $this->getView()->layout('Contacts.contact');
         }
```

Comment:
Is there any concern around how getView() works when you set properties in the controller, and then change the view class in beforeRender?. I didn't think of this until now when looking at the examples.